client/my-sites/stats/stats-purchase/tier-upgrade-slider/index.tsx

```diff
@@ -101,7 +107,7 @@ function TierUpgradeSlider( {
 					<h2>{ uiStrings.limits }</h2>
 					<p>{ lhValue }</p>
 				</div>
-				<div className="tier-upgrade-slider__step-callout right-aligned">
+				<div className={ secondaryCalloutClassnames }>
```

Comment:
You can use a condition to render this HTML or avoid it.
```
! secondaryCalloutClassnames && (
<div className={ secondaryCalloutClassnames }>
	<h2>{ uiStrings.price }</h2>
	<p ref={ infoReferenceElement }>{ rhValue }</p>
</div> )
```

This way, you won't have to add CSS just to hide something. It's better to avoid rendering it.

Reply:
```suggestion
				! secondaryCalloutIsHidden && ( <div className="tier-upgrade-slider__step-callout right-aligned">
```

I think you mean this way, isn't it? @grzegorz-cp

Reply:
Yes, we just need to wrap it with `{}`.

Reply:
Thanks for catching this. I'll get it fixed up today.